projects/storefrontstyles/scss/components/myaccount/_new-notification-preference.scss

```diff
@@ -0,0 +1,65 @@
+@import '../../theme';
+
+%cx-new-notification-preference{
+    .header{
+        width: $font-size-base * 14.375;
+        height: $font-size-base * 1.875;
+        font-weight: var(--cx-font-weight-semi);
+        font-size: $h3-font-size;
+        font-family: 'Open Sans';
+        line-height: $font-size-base * 1.8725;
+        margin-bottom: $font-size-base * 1.875;
+    }
+
+    .pref-info{
+        width: $font-size-base * 19.875;
+        height: $font-size-base * 1.375;
+        font-weight: var(--cx-font-weight-normal);
+        font-size: $h5-font-size;
+        font-family: 'Open Sans';
```

Comment:
please use $font-family-sans-serif
Use 'Open Sans' won't really take effect and will lead to pipeline error